### DISCUSSION TITLE
Nuclear Compensation: Lessons from Fukushima (Northwestern University Libraries)

### DISCUSSION THREAD

**Discussion by chrisdaaz** [thegetty/quire#52] [Show & Tell]
I'm nearly finished with _[Nuclear Compensation: Lessons from Fukushima](https://nuclear-compensation.northwestern.pub/)_, an edited monograph about nuclear disaster compensation issues published by Northwestern University Libraries. 

I've been really impressed by how Quire anticipates the desires of authors and publishers, such as providing both `grid` and `list` layouts for the table of contents, the availability of contributor/ bibliography short codes, and the ability to easily toggle specific pages for specific output formats. 

It was **especially** helpful to find the git repos of Getty-published Quire works to figure out how to implement specific elements (such as a copyright page). I hope to use Quire for more academic and educational projects in the future.

**Comment by geealbers** [comment]
Thanks for sharing @chrisdaaz! And thanks for pointing out a resource we have available, but I think don't do a good job of promoting, our own publications! Glad you found them and found them useful.

For others who might be interested ...

You can find many of our Quire publications in Getty's GitHub organization by searching for the "publications" and "quire" tags: https://github.com/search?q=org%3Athegetty+quire+publications. 

We try to make as many of our Quire project repositories public as we can (after [being careful what we include in the repos](https://github.com/thegetty/quire/discussions/69)). This gives us a public, scholarly record of changes made to the publication after its publication date, and a resource in developing other Quire projects. If you're interested in seeing what we did in a certain project you can look in the `layouts` directory where you'll see any templates or shortcodes we added or customized for the project, and in `static/css/custom.css` for all the custom styles. For the templates, we try to always add in all caps at the top of the file what it is we changed from the standard, default template. All Getty Quire projects start with the standard template that you get when you run `quire new`!
